src/style.css

```
@@ -1 +1,89 @@
-/* add styles here */
+:root {
+  --color: #3D4E61;
+  --background: #FFF;
+  --border: 1px solid #E1E7ED;
+  --box-sizing: border-box;
+  --border-radius: 4px;
+  --box-shadow: 0 1px 8px 0 #3d4e611a;
+  --box-shadow-hover: 0 3px 8px 0 rgba(61, 78, 97, 0.20);
+  --outline: none;
+}
+
+html {
+  font-weight: 300;
+  font-family: Avenir, sans-serif;
+}
+
+@font-face {
+  font-family: Avenir;
+  src: url(fonts/Avenir-Book.ttf);
+}
+
+.search-form__input--big,
+.search-form__input--small {
+  border-radius: var(--border-radius);
+  border: var(--border);
+  background: var(--background);
+  box-sizing: var(--box-sizing);
+  color: var(--color);
+}
+
+.search-form__input--big {
+  box-shadow: var(--box-shadow);
+  width: 100%;
+  height: 70px;
+  margin: 20px auto;
+  align-items: center;
+  display: flex;
```

```suggestion
  display: flex;
  align-items: center;
```
It's better to firstly decalte the "Flex" and only then it's properties